### PR TITLE
Add link to leaderboard

### PR DESCRIPTION
**Pull request by samjjc**
Add leaderboard link to main navigation bar

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 main:
   - title: "Posts"
     url: /posts/
+  - title: "Leaderboard"
```
Comment (JackyChiu) [inline _data/navigation.yml]: Should probably name it ice leaderboard bc it's under the context of `bread-gang` which doesn't directly imply icing

Comment (BenEmdon) [inline _data/navigation.yml]:
```suggestion
  - title: "Ice Leaderboard"
```

**Comment by BenEmdon** [comment]
@samjjc you are good to 🚢

**Comment by vikrambombhi** [comment]
@samjjc if u don't merge it in a hour ill do it for u

**Pull request merged**